core/lib/via_btc_client/examples/inscriber.rs

```diff
@@ -0,0 +1,75 @@
+use via_btc_client::types as inscribe_types;
+use via_btc_client::{inscriber::Inscriber, types::BitcoinNetwork, types::NodeAuth};
+
+use anyhow::{Context, Result};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // get the node url and private key from the environment
+
+    // export BITCOIN_NODE_URL="http://example.com:8332"
+    // export BITCOIN_PRV=example_wif
+
+    let url = std::env::var("BITCOIN_NODE_URL").context("BITCOIN_NODE_URL not set")?;
+    let prv = std::env::var("BITCOIN_PRV").context("BITCOIN_PRV not set")?;
+
+    let mut inscriber_instance = Inscriber::new(
+        &url,
+        BitcoinNetwork::Regtest,
+        NodeAuth::UserPass("via".to_string(), "via".to_string()),
+        &prv,
+        None,
+    )
+    .await
+    .context("Failed to create Inscriber")?;
+
+    println!(
```

Comment:
maybe we could use the `tracing` crate here. It will help understand how logs will look like in production
checkout the indexer example